index.js

```diff
@@ -1,4 +1,4 @@
-const { Config, TimeNode, StatsDB, Wallet, version } = require('./built');
+const { Config, TimeNode, StatsDB, Wallet, version } = require('./built/src');
```

Comment:
This change creates a weird folder structure and breaks some of the functionality.

![screen shot 2018-08-20 at 14 52 16](https://user-images.githubusercontent.com/9353642/44341604-ab39ce80-a488-11e8-9626-80569eb3ad5b.png)

Reply:
![screen shot 2018-08-20 at 14 53 09](https://user-images.githubusercontent.com/9353642/44341649-ce647e00-a488-11e8-887c-e24cd9d62ceb.png)